CHANGELOG.md

```diff
@@ -10,6 +10,9 @@ Remember to align the itemized text with the first line of an item within a list
 
 * Changes
   * The minimum NumPy version is now 1.24.
+  * `jax.numpy.ceil`, `jax.numpy.floor` and `jax.numpy.trunc` now return the output
+  of the same dtype as the input, i.e. no longer upcast integer inputs to floating
```

Comment:
Sorry, just noticed the formatting issue: lines 14-15 need to be indented two spaces for the markdown list to format correctly.

Reply:
Yes, I missed that, sorry